src/OpenLoco/src/Paint/PaintTrack.cpp

```diff
@@ -26,6 +26,91 @@ namespace OpenLoco::Paint
     static loco_global<uint8_t, 0x00113605E> _trackTunnel;
     static loco_global<uint8_t, 0x00522095> _byte_522095;
 
+    constexpr std::array<World::Pos3, 4> kStraightBoundingBoxOffsets = {
+        World::Pos3{ 2, 5, 0 },
+        World::Pos3{ 5, 2, 0 },
+        World::Pos3{ 2, 5, 0 },
+        World::Pos3{ 5, 2, 0 },
+    };
+    constexpr std::array<World::Pos3, 4> kStraightBoundingBoxSizes = {
+        World::Pos3{ 28, 22, 1 },
+        World::Pos3{ 22, 28, 1 },
+        World::Pos3{ 28, 22, 1 },
+        World::Pos3{ 22, 28, 1 },
+    };
+    constexpr std::array<std::array<uint32_t, 3>, 4> kStraightImageIndexOffsets = {
+        std::array<uint32_t, 3>{ TrackObj::ImageIds::Style0::straightBallastNE, TrackObj::ImageIds::Style0::straightSleeperNE, TrackObj::ImageIds::Style0::straightRailNE },
+        std::array<uint32_t, 3>{ TrackObj::ImageIds::Style0::straightBallastSW, TrackObj::ImageIds::Style0::straightSleeperSW, TrackObj::ImageIds::Style0::straightRailSW },
+        std::array<uint32_t, 3>{ TrackObj::ImageIds::Style0::straightBallastNE, TrackObj::ImageIds::Style0::straightSleeperNE, TrackObj::ImageIds::Style0::straightRailNE },
+        std::array<uint32_t, 3>{ TrackObj::ImageIds::Style0::straightBallastSW, TrackObj::ImageIds::Style0::straightSleeperSW, TrackObj::ImageIds::Style0::straightRailSW },
+    };
+    constexpr std::array<uint8_t, 4> kStraightBridgeEdges = {
+        0b0101,
+        0b1010,
+        0b0101,
+        0b1010,
+    };
+    constexpr uint8_t kStraightBridgeQuarters = 0xF;
+    constexpr std::array<uint8_t, 4> kStraightTunnelsEdges = {
+        0b0101,
+        0b1010,
+        0b0101,
+        0b1010,
+    };
+    constexpr std::array<uint16_t, 4> kStraightSegments = {
```

Comment:
Can anyone remember how the segments worked in RCT2?

Reply:
Are the the 3x3 grid of support-blocking segments?

Reply:
The bits (seem to) match the order of these segments: https://github.com/OpenRCT2/OpenRCT2/pull/3220#issuecomment-204704164